src/api/public/apidocs-new/paths/source_project_name_project_file.yaml

```diff
@@ -0,0 +1,60 @@
+get:
+  summary: Read a project file
+  description: |
+    Read a project file
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/file_name.yaml'
+    - in: query
+      name: meta
+      required: false
+      schema:
+        type: integer
+      description: Set to 1 to include _meta files
+      example: 1
+    - in: query
+      name: rev
+      required: false
+      schema:
+        type: integer
+      description: Revision number
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/project_meta.yaml'
+          example:
+            name: home:Admin
+            title: Endless Night
+            description: Test project
+            person:
+              userid: Admin
+              role: maintainer
+            repository:
+              name: openSUSE_Tumbleweed
+              path:
+                project: openSUSE.org:openSUSE:Factory
+                repository: snapshot
+              arch:
+                - x86_64
+    '400':
+      description: Wrong value for meta parameter.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            not boolean:
+              value:
+                code: 400
+                summary: "not boolean"
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/unknown_project.yaml'
+  tags:
+    - Sources - Projects
```

Comment:
I know this endpoint was listed in the card about `Sources - Projects` but I think it should belong to `Sources - Packages` as `/source/{project_name}/_project/{file_name}` is, in fact, the route to a file inside a package named `_project`

Reply:
Let's postpone the conversation about this. It's just a matter of which list we display the endpoint in. Something we can decide later when all the `source` endpoints are implemented.